### PR TITLE
[vcpkg] translate the README to french

### PR DESCRIPTION
**Pull request by x0rld**
**Describe the pull request**
I translated the README to french to help people

### CODE REVIEW

File: README_fr.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+```
+[vcpkg root]/scripts/buildsystems/vcpkg.cmake
+```
+
+### Vcpkg avec CLion
+
+ouvrez les paramètres de Toolchains 
+(File > Settings on windows et linux, CLion > Preference pour macOS ensuite allez dans CMake ).
+Dans Cmake options, ajouter la ligne suivante:
+
```
Comment (AntoineJT) [inline README_fr.md:233]:
```suggestion
Ouvrez les paramètres de Toolchains (File > Settings sur Windows et Linux, CLion > Preference pour macOS) et allez dans les paramètres CMake (Build, Execution, Deployment > CMake).
Finalement, dans `CMake options`, ajoutez la ligne suivante :
```

File: README_fr.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+```
+-DCMAKE_TOOLCHAIN_FILE=[vcpkg root]/scripts/buildsystems/vcpkg.cmake
+```
+
+Vous devrez le refaire pour chaque projet
```
Comment (AntoineJT) [inline README_fr.md:239]:
```suggestion
Malheureusement, vous devrez le refaire pour chaque projet.
```

File: README_fr.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+
+
+### Vcpkg en tant que sous module
```
Comment (AntoineJT) [inline README_fr.md:242]:
```suggestion
### Vcpkg en tant que sous-module
```

File: README_fr.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+quand vous utilisez vcpkg comme un sous module de votre projet, vous pouvez l'ajouter au CMakeLists.txt avant le premier appel de `project()`, au lieu d'utiliser `CMAKE_TOOLCHAIN_FILE` dans les paramètres d'appel de cmake
```
Comment (AntoineJT) [inline README_fr.md:244]:
```suggestion
Quand vous utilisez vcpkg comme un sous-module de votre projet, vous pouvez l'ajouter à votre CMakeLists.txt avant le premier appel de `project()`, au lieu d'utiliser `CMAKE_TOOLCHAIN_FILE` dans les paramètres d'appel de cmake.
```

File: README_fr.md [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+
+```cmake
+set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake
+  CACHE STRING "Vcpkg toolchain file")
+```
+
+Cela n'empêche pas d'utiliser `CMAKE_TOOLCHAIN_FILE` directement par la commande cmake mais ça permet de simplifier la configuration.
```
Comment (AntoineJT) [inline README_fr.md:251]:
```suggestion
Cela permettra toujours aux gens de ne pas utiliser vcpkg, en passant directement le CMAKE_TOOLCHAIN_FILE, mais cela rendra l'étape de configuration-construction légèrement plus facile.
```

File: README_fr.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+[getting-started:git]: https://git-scm.com/downloads
+[getting-started:cmake-tools]: https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools
+[getting-started:linux-gcc]: #installing-linux-developer-tools
+[getting-started:macos-dev-tools]: #installing-macos-developer-tools
+[getting-started:macos-brew]: #installing-gcc-on-macos
+[getting-started:macos-gcc]: #installing-gcc-on-macos
+[getting-started:visual-studio]: https://visualstudio.microsoft.com/
+[getting-started:manifest-spec]: docs/specifications/manifests.md
+
+# Tab-Completion/Auto-Completion
```
Comment (AntoineJT) [inline README_fr.md:264]:
```suggestion
# Tab-complétion/Auto-complétion
```

File: README_fr.md [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+
+`vcpkg` supporte l'auto-completion des commandes, nom de paquets, et options dans powershell et bash.
```
Comment (AntoineJT) [inline README_fr.md:266]:
```suggestion
`vcpkg` supporte l'auto-complétion des commandes, nom de paquets, et options dans powershell et bash.
```

File: README_fr.md [hunk 8]
```diff
@@ -0,0 +1,317 @@
...
+pour activer la tab-completion dans le shell de votre choix:
```
Comment (AntoineJT) [inline README_fr.md:267]:
```suggestion
Pour activer la tab-complétion dans le shell de votre choix :
```

File: README_fr.md [hunk 9]
```diff
@@ -0,0 +1,317 @@
...
+> .\vcpkg integrate powershell
+```
+
+ou
+
+```sh
+$ ./vcpkg integrate bash
+```
+
+suivant le shell qui est utiliser, redémarrer la console
```
Comment (AntoineJT) [inline README_fr.md:279]:
```suggestion
selon le shell que vous utilisez, puis redémarrez la console.
```

File: README_fr.md [hunk 10]
```diff
@@ -0,0 +1,317 @@
...
+
+# Exemples
+
+regarde la [documentation](doc/index.md) pour des instructions plus spécifiques ainsi que [l'installation et l'utilisation des paquets](docs/examples/installing-and-using-packages.md)
```
Comment (AntoineJT) [inline README_fr.md:283]:
```suggestion
Regardez la [documentation](doc/index.md) pour des instructions plus spécifiques ainsi que [l'installation et l'utilisation des paquets](docs/examples/installing-and-using-packages.md),
```

File: README_fr.md [hunk 11]
```diff
@@ -0,0 +1,317 @@
...
+[ajouter un paquet depuis un fichier zip](docs/examples/packaging-zipfiles.md),
```
Comment (AntoineJT) [inline README_fr.md:284]:
```suggestion
[ajouter un nouveau paquet depuis un fichier zip](docs/examples/packaging-zipfiles.md),
```

File: README_fr.md [hunk 12]
```diff
@@ -0,0 +1,317 @@
...
+et [ajouter un nouveau paquet depuis un repo GitHub](docs/examples/packaging-github-repos.md).
```
Comment (AntoineJT) [inline README_fr.md:285]:
```suggestion
et [ajouter un nouveau paquet depuis un dépôt GitHub](docs/examples/packaging-github-repos.md).
```

File: README_fr.md [hunk 13]
```diff
@@ -0,0 +1,317 @@
...
+
+La documentation est aussi disponible en ligne sur ReadTheDocs: <https://vcpkg.readthedocs.io/>!
```
Comment (AntoineJT) [inline README_fr.md:287]:
```suggestion
La documentation est aussi disponible en ligne sur ReadTheDocs : <https://vcpkg.readthedocs.io/> !
```

File: README_fr.md [hunk 14]
```diff
@@ -0,0 +1,317 @@
...
+
+regarde une démo de 4 minutes en anglais [video demo](https://www.youtube.com/watch?v=y41WFKbQFTw).
```
Comment (AntoineJT) [inline README_fr.md:289]:
```suggestion
Voir une [démo vidéo]((https://www.youtube.com/watch?v=y41WFKbQFTw) de 4 minutes en anglais.
```

File: README_fr.md [hunk 15]
```diff
@@ -0,0 +1,317 @@
...
+
+# Contribuer
+
+Vcpkg est un projet opensource, et évolue avec vos contributions
```
Comment (AntoineJT) [inline README_fr.md:293]:
```suggestion
Vcpkg est un projet open source, et évolue ainsi avec vos contributions. 
Voici quelques moyens pour vous d'y contribuer :
```

File: README_fr.md [hunk 16]
```diff
@@ -0,0 +1,317 @@
...
+
+
```
Comment (AntoineJT) [inline README_fr.md:295]:
```suggestion
```

File: README_fr.md [hunk 17]
```diff
@@ -0,0 +1,317 @@
...
+* [créer Issues][contributing:submit-issue] in vcpkg or existing packages
```
Comment (AntoineJT) [inline README_fr.md:296]:
```suggestion
* [Soumettre des Issues][contributing:submit-issue] sur vcpkg ou des paquets existants
```

File: README_fr.md [hunk 18]
```diff
@@ -0,0 +1,317 @@
...
+* [proposer des fix et des nouveaux paquets  que vous faites][contributing:submit-pr]
```
Comment (AntoineJT) [inline README_fr.md:297]:
```suggestion
* [Proposer des corrections et de nouveaux paquets][contributing:submit-pr]
```

File: README_fr.md [hunk 19]
```diff
@@ -0,0 +1,317 @@
...
+
+
+s'il vous plait referez-vous au [ guide de contribution](CONTRIBUTING.md) pour plus de détails.
```
Comment (AntoineJT) [inline README_fr.md:300]:
```suggestion
Veuillez vous référer au [guide de contribution](CONTRIBUTING.md) pour plus de détails.
```

File: README_fr.md [hunk 20]
```diff
@@ -0,0 +1,317 @@
...
+
+
+This project has adopted the [Microsoft Open Source Code of Conduct][contribuer:coc].
```
Comment (AntoineJT) [inline README_fr.md:303]:
```suggestion
Ce projet a adopté le [Code de Conduite Open Source de Microsoft][contribuer:coc].
```

File: README_fr.md [hunk 21]
```diff
@@ -0,0 +1,317 @@
...
+[contributing:submit-issue]: https://github.com/microsoft/vcpkg/issues/new/choose
+[contributing:submit-pr]: https://github.com/microsoft/vcpkg/pulls
+[contributing:coc]: https://opensource.microsoft.com/codeofconduct/
+[contributing:coc-faq]: https://opensource.microsoft.com/codeofconduct/
+
+# Licence
+  Le code du repository est sous licence [MIT](LICENSE.txt)
```
Comment (AntoineJT) [inline README_fr.md:310]:
```suggestion
Le code sur ce dépôt est distribué sous [licence MIT](LICENSE.txt).
```

File: README_fr.md [hunk 22]
```diff
@@ -0,0 +1,317 @@
...
+
+# Télémétrie
+
+vcpkg collecte des données pour nous aider à améliorer votre experience. Les données collecté par Microsoft sont anonymes.
```
Comment (AntoineJT) [inline README_fr.md:314]:
```suggestion
vcpkg collecte des données pour nous aider à améliorer votre expérience.
Les données collectées par Microsoft sont anonymes.
```

File: README_fr.md [hunk 23]
```diff
@@ -0,0 +1,317 @@
...
+vous pouvez désactiver la télémétrie en relançant le bootstrap-vcpkg avec l'argument `-disableMetrics`, passer l'argument `--disable-metrics` à chaque execution de vcpkg, ou en créant une variable d'environnement nommé `VCPKG_DISABLE_METRICS`;
```
Comment (AntoineJT) [inline README_fr.md:315]:
```suggestion
Vous pouvez désactiver la télémétrie en relançant le bootstrap-vcpkg avec l'argument `-disableMetrics`, passer l'argument `--disable-metrics` à chaque exécution de vcpkg, ou en créant une variable d'environnement nommée `VCPKG_DISABLE_METRICS`.
```

File: README_fr.md [hunk 24]
```diff
@@ -0,0 +1,317 @@
...
+
+Vous trouverez plus d'informations à propos de la télémétrie dans vcpkg sur cette [page](docs/about/privacy.md)
```
Comment (AntoineJT) [inline README_fr.md:317]:
```suggestion
Vous trouverez plus d'informations à propos de la télémétrie dans vcpkg sur cette [page](docs/about/privacy.md).
```

**Review by AntoineJT — CHANGES_REQUESTED** [review]
Here are some changes : the most of them are typo fixing, other are wording changes to better stick with the english readme, etc.

**Review by x0rld — COMMENTED** [review]
it's done just some typos to fix

File: README.md [hunk 25]
```diff
@@ -2,7 +2,7 @@
 
 [中文总览](README_zh_CN.md)
 [Español](README_es.md)
-
+[Français](README_fr.md)
```
Comment (JackBoosY) [inline README.md:5]:
```suggestion
[Français](README_fr.md)

```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: README_fr.md [hunk 26]
```diff
@@ -0,0 +1,311 @@
...
+Si vous êtes intéressé par le futur de vcpkg, regardez le guide du [manifeste][getting-started:manifest-spec] !
+C'est une fonctionnalité expérimentale et possiblement boguée, donc essayez d'[ouvrir des issues](contribuer:envoyer-une-issue) !
+
+# Introduction: Windows
+Prérequis :
+  - Windows 7 ou plus
+  - [Git][getting-started:git]
+  + [Visual Studio][getting-started:visualstudio] 2015 mise à jour 3 où plus récente avec le pack de langue Anglais
+
+Premièrement, téléchargez et lancer le fichier bootstrap-vcpkg; il peut être installé n'importe où mais il est recommandé d'utiliser vcpkg pour des projets CMake, Nous recommandons ces chemins `C:\src\vcpkg` ou `C:\dev\vcpkg`, sinon vous pourriez avoir des problèmes de chemin pour certaines compilations.
```
Comment (vicroms) [inline README_fr.md:60]:
```suggestion
Premièrement, téléchargez et lancer le fichier bootstrap-vcpkg; il peut être installé n'importe où mais il est recommandé d'utiliser vcpkg pour des projets CMake. Nous recommandons ces chemins `C:\src\vcpkg` ou `C:\dev\vcpkg`, sinon vous pourriez avoir des problèmes de chemin pour certaines compilations.
```

**Review by vicroms — COMMENTED** [review]
Just a small nit (comma to period), feel free to disregard since I don't know French.

File: README_fr.md [hunk 27]
```diff
@@ -0,0 +1,311 @@
...
+```
+
+Vous pouvez aussi chercher la bibliothèque dont vous avez besoin avec l'argument `search`:
+
+```cmd
+> .\vcpkg\vcpkg search [terme de recherche]
+```
+
+
+Pour utiliser vcpkg avec Visual Studio, lancez cette commande (pourrait nécessiter une élévation en administrateur)
```
Comment (Lectem) [inline README_fr.md:81]: "Élévation en administrateur" I would have written "d'être lancée avec les droits administrateurs". The term "élévation" looks weird to me in French, even though it's a direct translation of the English term

File: README_fr.md [hunk 28]
```diff
@@ -0,0 +1,311 @@
...
+
+```cmd
+> .\vcpkg\vcpkg integrate install
+```
+
+Après ça, vous pouvez créer un nouveau projet n'utilisant pas CMake (ou en ouvrir un préexistant).
```
Comment (Lectem) [inline README_fr.md:87]: Suggest `après ça` => `ensuite`

File: README_fr.md [hunk 29]
```diff
@@ -0,0 +1,311 @@
...
+Pour utiliser vcpkg en dehors d'un IDE, vous pouvez utiliser le fichier de toolchain :
+
+```cmd
+> cmake -B [dossier de build] -S . -DCMAKE_TOOLCHAIN_FILE=[chemin vers vcpkg]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [dossier de build]
+```
+
+Avec CMake, vous devrez utiliser `find_package` notamment, et autres, pour compiler.
+
+Regardez la [section CMake](#utiliser-vcpkg-avec-cmake) pour plus d'information, notamment utiliser CMake avec un IDE.
```
Comment (Lectem) [inline README_fr.md:147]: "Regardez" => "Lisez"?

File: README_fr.md [hunk 30]
```diff
@@ -0,0 +1,311 @@
...
+
+Pour les autres éditeurs, y compris Visual Studio Code regardez le [guide d'intégration][getting-started:integration].
```
Comment (Lectem) [inline README_fr.md:149]: "Regardez" => "Lisez"?

File: README_fr.md [hunk 31]
```diff
@@ -0,0 +1,311 @@
...
+```json
+{
+  "cmake.configureSettings": {
+    "CMAKE_TOOLCHAIN_FILE": "[vcpkg root]/scripts/buildsystems/vcpkg.cmake"
+  }
+}
+```
+### Vcpkg avec des projets CMake Visual Studio
+
+Ouvrez les paramètres CMake, et ajouter le chemin ci-dessous à `CMake toolchain file` :
```
Comment (Lectem) [inline README_fr.md:218]: Ajoutez

File: README_fr.md [hunk 32]
```diff
@@ -0,0 +1,311 @@
...
+
+```sh
+$ ./vcpkg integrate bash
+```
+
+selon le shell que vous utilisez, puis redémarrez la console.
+
+# Exemples
+
+Regardez la [documentation](doc/index.md) pour des instructions plus spécifiques ainsi que [l'installation et l'utilisation des paquets](docs/examples/installing-and-using-packages.md),
```
Comment (Lectem) [inline README_fr.md:277]: "Regardez" => "Lisez"?

File: README_fr.md [hunk 33]
```diff
@@ -0,0 +1,311 @@
...
+[ajouter un nouveau paquet depuis un fichier zip](docs/examples/packaging-zipfiles.md),
+et [ajouter un nouveau paquet depuis un dépôt GitHub](docs/examples/packaging-github-repos.md).
+
+La documentation est aussi disponible en ligne sur ReadTheDocs : <https://vcpkg.readthedocs.io/> !
+
+Voir une [démo vidéo]((https://www.youtube.com/watch?v=y41WFKbQFTw) de 4 minutes en anglais.
```
Comment (Lectem) [inline README_fr.md:283]: Voir => regardez

File: README_fr.md [hunk 34]
```diff
@@ -0,0 +1,311 @@
+# Vcpkg: Vue d'ensemble
+
+[中文总览](README_zh_CN.md)
+[Español](README_es.md)
+
+Vcpkg vous aide à gérer vos bibliothèques C et C++ sur Windows, Linux et MacOS.
+L'outil et l'écosystème sont en évolution constante, et nous apprécions vos contributions!
+
+Si vous n'avez jamais utilisé vcpkg, ou si vous essayez d'utiliser vcpkg, regardez notre [introduction](#introduction) pour comprendre comment l'utiliser.
```
Comment (Lectem) [inline README_fr.md:9]: "Regardez" => "Lisez"?

**Review by Lectem — COMMENTED** [review]
A few nitpicks on things that look weird to me as a french person (maybe not to someone from Québec?), but otherwise looks good

File: README_fr.md [hunk 35]
```diff
@@ -0,0 +1,311 @@
...
+    - [Contribuer](#contribuer)
+    - [Licence](#licence)
+    - [Télémétrie](#Télémétrie)
+    
+# Introduction
+
+Premièrement, suivez le guide d'introduction [Windows](#Introduction:-Windows), ou [macOS et Linux](#Unix), en fonction de vos besoins.
+
+Pour plus d'information, regardez [utiliser des paquets][getting-started:utiliser-un-paquet].  
+Si la bibliothèque dont vous avez besoin n'est pas présente dans la liste, vous pouvez [ouvrir une issue sur le repo github](contribuer:faire-une-issue) où l'équipe de vcpkg et la communauté peut le voir, et possiblement ajouter le port de vcpkg.
```
Comment (Asharas) [inline README_fr.md:47]: "peut" -> "peuvent"
3rd-person plural

File: README_fr.md (hunk 34 above)
Comment (x0rld) [reply README_fr.md:9]: thanks for the review

File: README_fr.md (hunk 35 above)
Comment (x0rld) [reply README_fr.md:47]: thanks for the review

**Comment by BillyONeal** [comment]
Je m'appelle Bill. Merci!  ... something something French.

Also thanks to Olivier Giroux who double checked: https://twitter.com/__simt__/status/1322026346256560128

**Pull request merged**